### ISSUE TITLE
Error when using variables in Saved Replies 

### ISSUE THREAD

**Issue by nerdalertdk** [freescout-helpdesk/freescout#774]
Hi

When using variables in saved replies they don't get converted.

We tested it with these 

```
{%user.fullName%}
{%user.photoUrl%}
```

but they just print out 

```
{%user.fullName%}
{%user.photoUrl%}
```

in the mail

**Comment by freescout-helpdesk** [comment]
Variables are replaced with values in a moment you select Saved Reply from the dropdown:

![2020-10-02_11-26-21](https://user-images.githubusercontent.com/40499291/94903203-1ea0cd80-04a2-11eb-8816-fe540c1b0279.png)

**Comment by nerdalertdk** [comment]
Hi

For me it seams the first use give the variables but the next gives the correct data

https://share.getcloudapp.com/llu2zPOq

**Comment by freescout-helpdesk** [comment]
We could not reproduce the issue.

**Comment by nerdalertdk** [comment]
Hi,
seams to be fixed in 1.6.6, could have been an cache problem, but we did get in on both on an Windows and Mac
